src/main/fc/cli.c

```diff
@@ -2780,7 +2780,7 @@ static void printResource(uint8_t dumpMask)
             const ioTag_t ioTagDefault = *((const ioTag_t *)defaultConfig + resourceTable[i].offset + index);
 
             bool equalsDefault = ioTag == ioTagDefault;
-            const char *format = "resource %s %d %c%02dn";
+            const char *format = "resource %s %d %c%02d\n";
```

Comment:
this should not have a `\n` on the end of it as it passed to `cliDefaultPrintLinef` which already takes care of the LF + CR.